pkg/api/config.go

```diff
@@ -21,8 +21,9 @@ type Config struct {
 	ConfigStorageAccount   string `json:"configStorageAccount,omitempty"`
 	RegistryStorageAccount string `json:"registryStorageAccount,omitempty"`
 
-	Certificates CertificateConfig `json:"certificates,omitempty"`
-	Images       ImageConfig       `json:"images,omitempty"`
+	Certificates     CertificateConfig `json:"certificates,omitempty"`
+	RootDockerConfig []byte            `json:"rootDockerConfig,omitempty"`
```

Comment:
s/RootDockerConfig/ImagePullSecret/ ? to match existing ones?

Reply:
I still think its part of `Images` config struct, and name Should change as its not dockerConfig, its Image pull secret. Docker config is `/etc/docker/daemon.json` and all subfolder for it.